modules/core/src/main/java/org/apache/ignite/IgniteSystemProperties.java

```diff
@@ -1012,10 +1012,7 @@
     /**
      * When set to {@code true} fields are written by BinaryMarshaller in sorted order. Otherwise
      * the natural order is used.
-     * <p>
-     * @deprecated Should be removed in Apache Ignite 2.0.
      */
```

Comment:
Can we add a notice that it needs to be made default in Apache Ignite 3.0?

Reply:
Let's notice it in IGNITE-9702

Reply:
We can't add notice for 2.x in the ticket filed against 3.0. I think it should be noted in source, you can include ticket link if you like

You can also file a new ticket for 2.x which will introduce sortedField boolean into BinaryConfiguration, if you like to postpone it.
Or just add the notice now.